commons/prod-analytics/configmaps/flyway-application-schema.yaml

```
@@ -0,0 +1,109 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: common-flyway-application-schema
+  namespace: prod-analytics
+data:
+  V1__application_Schema.sql: |-
+    CREATE SCHEMA IF NOT EXISTS application;
+
+    GRANT USAGE ON SCHEMA application TO
+      interop_be_application_audit_analytics_writer_${NAMESPACE};
```

We're going to deploy KPI in prod by using the new users naming.
Because of this, it is needed to edit the migrations whenever a user is referenced.
See this [PR](https://github.com/pagopa/interop-analytics-deployment/pull/97/files).
```suggestion
      ${NAMESPACE}_application_audit_analytics_writer_user;
```